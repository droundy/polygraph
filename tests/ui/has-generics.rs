polygraph::schema!{
    type MySchema;
    pub struct Struct<T>(T);
}

fn main() {
}
