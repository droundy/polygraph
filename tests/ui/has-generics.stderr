error: schema! does not support generic types.
 --> $DIR/has-generics.rs:3:22
  |
3 |     pub struct Struct<T>(T);
  |                      ^^^
